spec/acceptance/enum_traits_spec.rb

```diff
@@ -113,6 +113,211 @@ def each(&block)
         expect(task.status).to eq(trait_name)
       end
     end
+
+    context 'when prefix is used' do
+      context 'when values are a hash' do
```

Comment:
Do prefix and suffixes interact differently with a hash vs. list, or is there something specific we're trying to cover by testing against both? If not, I think we could choose one or the other to test with, since we've already got coverage of the various enumerable types.

Reply:
With the current form of the code, no they don't behave differently, however with hash being passed in one of the tests (enum_for_trait_spec.rb:60) was failing as the keynames of the hash didn't match with the prefix, suffix options, so i chose to leave the 2 different tests in (for now at least) to ensure i pushed a solution that worked for all implementation types of the api